client/main.lua

```diff
@@ -444,6 +444,138 @@ function ReturnOwnedPoliceMenu()
 end
 -- End of Police Code
 
+-- Start of Mechanic Code
```

Comment:
Instead of declaring function per job, it's much better to add arguments on the function and declare a job-specific function for each. It will also impact the script memory resources.